SnowWatch.js

```diff
@@ -47,10 +47,12 @@ SnowWatch.prototype.isSnowyEnough = function(forecast) {
     + ", minProb=" + this.precipProbabilityMin
     + ", enough? " + (forecast.precipProbability >= this.precipProbabilityMin));
   return (
-        forecast.precipType == 'snow' 
-      ||  forecast.precipType == 'sleet'
+    (
+      (forecast.precipType == 'snow' || forecast.precipType == 'sleet')
```

Comment:
probably don't need another set of parens. since any of them can be true in any order, not a blocker.